packages/api-client/src/v2/features/app/App.vue

```diff
@@ -142,27 +147,40 @@ const environment = computed<XScalarEnvironment>(() => {
 
 /** Generate the theme style tag for dynamic theme application. */
 const themeStyleTag = computed(() => {
-  const themeId = workspaceStore.workspace['x-scalar-theme']
+  if (store.value === null) {
+    return ''
+  }
+
+  const themeId = store.value.workspace['x-scalar-theme']
 
   if (!themeId) return ''
 
   return `<style>${getThemeStyles(themeId)}</style>`
 })
 
 /** Width of the sidebar, with fallback to default. */
-const sidebarWidth = computed(
-  () =>
-    workspaceStore.workspace['x-scalar-sidebar-width'] ?? DEFAULT_SIDEBAR_WIDTH,
-)
+const sidebarWidth = computed(() => {
+  if (store.value === null) {
+    return DEFAULT_SIDEBAR_WIDTH
+  }
+  return (
+    store.value.workspace['x-scalar-sidebar-width'] ?? DEFAULT_SIDEBAR_WIDTH
+  )
+})
```

Comment:
```suggestion
const sidebarWidth = computed(() =>  
  store.value?.workspace?.['x-scalar-sidebar-width'] ?? DEFAULT_SIDEBAR_WIDTH 
)
```
more efficient shorthand

Reply:
I waited YEARS for these to be added to typescript. You're damn right we're gunna use em!!

https://github.com/microsoft/TypeScript/issues/16